roles/core/pxe_stack/templates/menu.ipxe.j2

```diff
@@ -103,13 +103,10 @@ iseq ${platform} efi && goto bootdisk-efi || goto bootdisk-legacy
     chain http://${next-server}/preboot_execution_environment/bin/${arch}/grub2_efi_autofind.img || shell
     exit 1
 
-:memtest
-echo Booting Memtest86+
-boot http://${next-server}/preboot_execution_environment/bin/memtest86+-5.01.bin || shell
-
-:minimal_live_linux
-echo Booting Minimal Live Linux
-chain http://${next-server}/preboot_execution_environment/equipment_ipxe_configurations/minimal.ipxe
-
-
+#:memtest
```

Comment:
Why do you drop this?

Reply:
Doesnt work right now. Should be reworked. So commented for now.